package/libs/openssl/Makefile

```diff
@@ -154,6 +155,11 @@ ifndef CONFIG_OPENSSL_WITH_SRP
   OPENSSL_OPTIONS += no-srp
 endif
 
+ifdef CONFIG_OPENSSL_OPTIMIZE_SPEED
+  TARGET_CFLAGS += -O3
+  TARGET_CFLAGS := $(filter-out -Os,$(TARGET_CFLAGS))
```

Comment:
Minor nit... you should be able to combine these lines as:

`TARGET_CFLAGS := $(filter-out -Os,$(TARGET_CFLAGS)) -O3`

Reply:
Fixed, thanks!